src/main/resources/mw_messages_en.properties

```diff
@@ -18,6 +18,7 @@ invalid.entryid.for.study=Invalid Entry ID for Study.
 new.gid.matches.old.gid=New GID matches GID of entry to be replaced.
 
 #Germplasm List Service
+list.not.found="List not found"
```

Comment:
Not sure why we are using "" for these messages @lik-83

Reply:
I added them by mistake. I'm gonna remove the quotes.